corehq/apps/reminders/models.py

```diff
@@ -1973,3 +1973,4 @@ def update_count(self, increase_by=1):
         EmailUsage.objects.filter(pk=self.pk).update(count=models.F('count') + increase_by)
 
 
+from corehq.apps.reminders import signals
```

Comment:
@gcapalbo I'm not very familiar with django's signals - what purpose does this "unused" import provide?  Tests fail without it, so it's "used" in some way.